tools/mtouch/error.cs

```diff
@@ -30,7 +30,7 @@ namespace Xamarin.Bundler {
 	//					MT0019	Only one --[log|install|kill|launch]dev or --[launch|debug]sim option can be used.
 	//					MT0020	The valid options for '{0}' are '{1}'.
 	//					MT0021	Cannot compile using gcc/g++ (--use-gcc) when using the static registrar (this is the default when compiling for device). Either remove the --use-gcc flag or use the dynamic registrar (--registrar:dynamic).
-	//					MT0022  The options '--unsupported--enable-generics-in-registrar' and '--registrar' are not compatible.
+	//					MT0022  <unused>
```

Comment:
Not _quite_ right, we need to avoid re-using those (at least too quickly) since they would confuse anyone googling for help.

I think we should also update (non visible comment?) the docs/website/mtouch-errors.md to mention the error is not produced anymore.